go-controller/pkg/node/bridgeconfig/bridgeflows.go

```diff
@@ -385,7 +385,7 @@ func (b *BridgeConfiguration) flowsForDefaultBridge(extraIPs []net.IP) ([]string
 					nodetypes.DefaultOpenFlowCookie, netConfig.V4MasqIPs.ManagementPort.IP.String()))
 
 			dftFlows = append(dftFlows,
-				fmt.Sprintf("cookie=%s, priority=250, table=2, ip, pkt_mark=%s, "+
+				fmt.Sprintf("cookie=%s, priority=150, table=2, ip, pkt_mark=%s, "+
```

Comment:
wont this need to be conditional on if bgp isolation is enabled? maybe put a todo or something and say what priority it should be without isolation?

Reply:
no even without BGP it needs to be at 150. The fix that was merged last week for fixing flows broken UDN to UDN nodeport isolation in some sense - its just that there were no tests so we never caught it. so this priority is required regardless of whether the network is advertised or not.

Reply:
today for udns and bgp udn pod -> same node for other networks is isolated and different node is allowed - this inconsistency exists :(